src/components/o-modal/index.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from '../../js/prop-types';
+import BaseComponentGlobal from '../../js/abstract/base-component-global';
+import defineOnce from '../../js/define-once';
+import styles from './index.scss';
+import template from './_template';
+
+class AXAModal extends BaseComponentGlobal {
+  static tagName = 'axa-modal';
+  static propTypes = {
+    open: PropTypes.bool,
+  };
+
+  set open(value) {
+    this.setAttribute('open', value);
+  }
+
+  get open() {
+    return this.getAttribute('open');
+  }
+
+  closeModal() {
+    this.removeAttribute('open');
+  }
+
+  closeModalByClickingOutside(event) {
+    if (event.target === this.querySelector('.js-modal')) this.closeModal();
```

Comment:
@raphaellueckl i think this is not in the ESLint rules, therefore you dont have to change it, but i highly suggest to always use brackets. This can lead to issues in future.